openeo/processes/generator.py

```diff
@@ -0,0 +1,163 @@
+import argparse
+import keyword
+import sys
+import textwrap
+from pathlib import Path
+from typing import Union, List, Iterator
+
+from openeo.processes.parse import Process, parse_all_from_dir
+
+
+class PythonRenderer:
```

Comment:
Minor: this utility could be kept out of the source tree, to avoid packaging it in the wheel? Just to avoid confusing users.

Reply:
will move this to `openeo.internal`